src/samples/Uno.Material.Samples/Uno.Material.Samples.Shared/Content/Controls/ModalStandardBottomSheetSamplePage.xaml.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Navigation;
+
+namespace Uno.Material.Samples.Content.Controls
+{
+	public sealed partial class ModalStandardBottomSheetSamplePage : Page
+	{
+		public ModalStandardBottomSheetSamplePage()
+		{
+			this.InitializeComponent();
+			this.SeeCodeBehindButton.Content = GetCodeBehindSource().Replace("\t", "    ");
+			this.SeeDataTemplateCodeButton.Content = GetDataTemplateCodeSource().Replace("\t", "    ");
+		}
+
+		private string GetCodeBehindSource()
+		{
+			return
+@"private void Button_Clicked(object sender, RoutedEventArgs e)
+{
+		ModalStandardBottomSheet.IsOpened = true;
+}";
+		}
+
+		private string GetDataTemplateCodeSource()
+		{
+			return
+@"<Page.Resources>
```

Comment:
@Xiaoy312 is there a way to link to this via xaml name or something? or we really need to duplicate the xaml?

Reply:
Not with the current implementation. But, nothing stops us from defining the ContentTemplate inline (within the xamldisplay)